teku/src/main/java/tech/pegasys/teku/cli/subcommand/internal/validator/tools/ConsoleAdapter.java

```diff
@@ -16,10 +16,18 @@
 import java.io.Console;
 
 public class ConsoleAdapter {
-  private Console console = System.console();
+  private final Console console = System.console();
 
+  @SuppressWarnings("SystemConsoleNull") // https://errorprone.info/bugpattern/SystemConsoleNull
```

Comment:
This came up after adding `final` to the console variable above.
It has nothing to do with the `FieldCanBeFinal` validation itself but it triggered another EP check and mentioned in the comment.

Reply:
Although this change makes sense given the imminent JDK 22 update, I would like to know that we have tested this using JDK 22 to be sure. Otherwise, we may think that everything is fine but won't find out until we upgrade our testing to JDK 22.

Reply:
tested and feature still works as expected